anago

```diff
@@ -403,25 +403,41 @@ generate_release_notes() {
   logecho -n "Checkout master branch to make changes: "
   logrun -s git checkout master || return 1
 
-  # The fetch and rebase before editing CHANGELOG.md
+  # The fetch and rebase before editing $CHANGELOG_FILE
   # avoids merge conflicts if another release cut
   # completed while we were building this one.
-  logecho -n "Fetch origin/master to get latest CHANGELOG.md: "
+  logecho -n "Fetch origin/master to get latest $CHANGELOG_FILE: "
   logrun -s git fetch origin master || return 1
-  logecho -n "Rebase on origin/master before editing CHANGELOG.md: "
+  logecho -n "Rebase on origin/master before editing $CHANGELOG_FILE: "
   logrun -s git rebase origin/master || return 1
 
-  logecho "Insert $RELEASE_VERSION_PRIME notes into CHANGELOG.md..."
+
+# TODO: Is there any value/risk/benefit to creating a symlink to CHANGELOG.md
```

Comment:
Users may be a bit confused at the first sight when they see CHANGELOG.md and CHANGELOG-1.NewestVersion.md are both the changelog of the newest version ?

Reply:
aliasing CHANGELOG.md to something unstable leads to links that break... I'd prefer just the release-branch-specific files